components/measurement/nettests/VanillaTor.js

```diff
@@ -10,16 +10,7 @@ import { MdTimelapse } from 'react-icons/md'
 
 import AccessPointStatus from '../AccessPointStatus'
 
-const VanillaTorDetails = ({ measurement, render }) => {
-  const testKeys = measurement.test_keys
-  const isOK = testKeys.success === true
-  const isAnomaly = testKeys.success === false
-  const torVersion = testKeys.tor_version
-  const torLog = testKeys.tor_log
-  const torProgress = testKeys.tor_progress
-  const isFailure = testKeys.error !== null
-  const failure = testKeys.failure
-
+const VanillaTorDetails = ({ measurement, render, isAnomaly, isFailure }) => {
```

Comment:
We only have 2 statuses for the VanillaTor screen - 'anomaly' and 'reachable' - probably we want to add 'failure' as well?

Reply:
Yeah we should ideally use the `isFailure` flag as well and implement the failure screen in those cases. We probably could even make this into something general purpose to use in most measurement pages. It's basically just a matter of coloring the header differently and showing the failure screen.